docs/actions-scripts.md

```diff
@@ -33,17 +112,17 @@ We currently package version 16.1, with `datacheck`, `safetab`, and `safecount`
 
 As Stata is a commercial product, a license key is needed to use it. 
 
-#### If you are a member of the opensafely GitHub organisation
+#### If you are a member of the OpenSAFELY GitHub organisation
 * If you are using Windows, then the `opensafely` command line software will
-automatically use the OpenSAFELY stata license. 
-* If you are using macOS:
+automatically use the OpenSAFELY Stata license. 
+* If you are using MacOS:
    1. Download and install [GitHub's command-line tool (`gh`)](https://cli.github.com/)
    2. Run `gh auth login --web`. Select the "HTTPS" option, and follow the instructions
   * The `opensafely` command line software will now automatically use the OpenSAFELY Stata license
 
 #### All other external users
 
-If you are not a member of the opensafely GitHub organisation, you must provide your own **Stata/MP** license. Unfortunately other Stata flavours are not yet supported; [let us know](how-to-get-help.md) if this is a problem.
+If you are not a member of the `opensafely GitHub organisation, you must provide your own **Stata/MP** license. Unfortunately other Stata flavours are not yet supported; [let us know](how-to-get-help.md) if this is a problem.
```

Comment:
"`opensafely" needs closing backtick

Reply:
or [OpenSAFELY GitHub organisation](https://github.com/opensafely) ?

Reply:
I didn't intentially change this, will revert